content/management/index.mdx

```diff
@@ -8,20 +8,125 @@ desc: "There are dedicated APIs for the resource management. Use the Configurati
 
 # Introduction
 
-LiveChat Platform provides multiple APIs for managing resources.
+The Configuration API allows you to manage resources such as **agents, auto access, bots, groups, properties, and webhooks**. You can use it to build advanced integrations as well as alter the configuration of your resources directly via the API.
 
-## New Configuration APIs
+## Use cases
 
-Configuration API is a service for storing configuration of license. It allows you to set up custom properties, bot agents and webhooks. You'll be using this API together with [Messaging APIs](../messaging/).
+Here are some of the most common use cases of the Configuration API:
 
-<SectionLink to={"/management/configuration-api/"}>
-  Configuration API Reference
+- Build integrations that connect LiveChat with other services via **webhooks**. React to events happening in the chat.
+- Create automation solutions with the use of **bots**. Connect LiveChat with various AI solutions.
+- Use **properties** to store information about chats, threads, or events.
+
+## API versioning
+
+The LiveChat Configuration API adopts the following versioning convention:
+
+- **stable**
+
+  The latest stable version we encourage developers to use. You can only expect bug fixes; no new new functionalities or breaking changes will be introduced.
+
+- **developer preview (dev preview)**
+
+  The unstable version that provides a preview of the upcoming functionalities. **It's not open for public use by default.** If you want to use it, contact us at developers@livechat.com or <a href="#open-chat" onClick={openChatWindow}>ask on the chat</a>, and we'll give you access.
+
+- **legacy**
+
+  The old version we still support. We highly encourage you to switch to the latest stable version.
+
+**Currently:**
+
+|                 | Configuration API |
+| --------------- | ----------------- |
+| **stable**      | 3.3               |
+| **dev preview** | 3.4               |
+| **legacy**      | 2.0, 3.1, 3.2     |
+
+<Warning>
+
+Currently, Configuration API v2 and v3 cover different resources. However, if you're still using v2, we encourage you to switch to v3 as more and more resources are being migrated to v3.
+
+</Warning>
+
+# Resources
+
+## Agents
+
+An agent is a type of user who communicates with customers. You can see the agent data structure [here](/messaging/agent-chat-api/#agent). The Configuration API allows for multiple agent-related actions, including creating, updating, deleting, and suspending agents.
+
+<SectionLink to={"/management/configuration-api/#agents"}>
+  See agent methods
+</SectionLink>
+
+## Auto access
+
+Auto access is a functionality that allows for assigning customers to dedicated groups based on the URL or the customer's geolocation.
```

Comment:
>to dedicated groups based on
based on different rules like a URL of the website or the customer's geolocation.

In fact, there are more rules to set, but we don't need to mention all of them here.

How about describing full context here? Because a developer has no idea what is customer group and why they would want to set it (automatically).

>...When such customer starts a chat, the chat will be routed to the specific group.

Reply:
That might be a good idea